hack/update-mocks.sh

```diff
@@ -24,6 +24,6 @@ do
   MOCK_DIR=$(echo $i | awk -F: '{print $1}')
   MOCK_NAME=$(echo $i | awk -F: '{print $2}')
   cd ${REPO_ROOT}/${MOCK_DIR}
-  mockery -name ${MOCK_NAME}
+  mockery --name ${MOCK_NAME}
```

Comment:
Why?

```
mockery
Use -name to specify the name of the interface or -all for all interfaces found
```

Reply:
The version that gets installed via `brew install vektra/tap/mockery` is `2.1.0`. From that version's help string: 
```
--name string            name or matching regular expression of interface to generate mock for
```

Reply:
They changed to viper for v2 in [this commit](https://github.com/vektra/mockery/commit/1549900a3e99e3d13c9585005cbe0922b41e5d35)

Reply:
In Golang `flag`, double dash is same as single dash, though golang prefers single.

Reply:
@jonnylangefeld - Thanks for sharing that!

Reply:
hmm 🤔  I don't think it's the same 😄 
run from within the `eventbus/driver` directory:
```
mockery -name Driver
Error: unknown command "Driver" for "mockery"
Run 'mockery --help' for usage.
```
It works with two dashes.

`make codegen` also fails with the newer version of mockery with only a single dash.

Reply:
That is because you are using new version of mockery (as you said, it changed to use `viper`, where double dash does not equal to single dash any more).